dmn/src/test/java/io/camunda/zeebe/dmn/DmnEvaluatedDecisionsTest.java

```diff
@@ -143,4 +148,66 @@ void shouldReturnPartialResultIfEvaluationFails() {
     assertEquality(evaluatedDecisions.get(0).decisionOutput(), "'Jedi'");
     assertEquality(evaluatedDecisions.get(1).decisionOutput(), "null");
   }
+
+  @Nested
+  @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+  @DisplayName("If successfully evaluated, the decision")
+  class DecisionTypeTests {
+
+    private static final String DECISION_TYPES_DRG = "/drg-decision-types.dmn";
+
+    Stream<DecisionTest> decisions() {
+      return Stream.of(
+          new DecisionTest("decision_table", DecisionType.DECISION_TABLE, "'okay'"),
+          new DecisionTest("literal_expression", DecisionType.LITERAL_EXPRESSION, "'okay'"),
+          new DecisionTest("context", DecisionType.CONTEXT, "{'is':'okay'}"),
+          new DecisionTest("invocation", DecisionType.INVOCATION, "'okay'"),
+          new DecisionTest("list", DecisionType.LIST, "['okay']"),
+          new DecisionTest("relation", DecisionType.RELATION, "[{'is':'okay'}]"));
+    }
```

Comment:
👍 Nice usage of a parameterized tests

Reply:
We be awesome to have named parameters :smile: